inc/class-cron.php

```diff
@@ -81,28 +98,43 @@ public function buildTheQueue() {
 	 *
 	 */
 	public function scheduleEvents() {
-		$b_timestamp = wp_next_scheduled( 'cwp_cron_build_hook' );
-
-		if ( ! $b_timestamp ) {
-			wp_schedule_event( time(), 'daily', 'cwp_cron_build_hook' );
-		}
 
 		$m_timestamp = wp_next_scheduled( 'cwp_cron_notify_hook' );
 
 		if ( ! $m_timestamp ) {
-			wp_schedule_event( time(), 'cwp_five_minutes', 'cwp_cron_notify_hook' );
+			wp_schedule_event( time(), 'cwp_two_minutes', 'cwp_cron_notify_hook' );
 		}
 	}
 
-	public function unScheduleEvents( $hook ){
+	public function unScheduleEvents( $hook ) {
+		if ( empty( $hook ) ) {
+			return;
+		}
 		$prefix = 'cwp_';
-		$sub = substr( $hook, 0, 4 );
+		$sub    = substr( $hook, 0, 4 );
 
-		// restrict d-registering events to our own
+		// restrict de-registering events to our own
 		if ( 0 === strcmp( $prefix, $sub ) ) {
 			$timestamp = wp_next_scheduled( $hook );
 			wp_unschedule_event( $timestamp, $hook );
 		}
 	}
+
+	public function scheduleEventCustomInterval( $hook, $interval ) {
```

Comment:
This line causes the following warning: 
`( ! ) Warning: Missing argument 2 for BCcampus\Cron::scheduleEventCustomInterval(), called in /var/www/public/eypd/wp-includes/class-wp-hook.php on line 286 and defined in /var/www/public/eypd/wp-content/plugins/custom-wp-notify/inc/class-cron.php on line 123`